docs/docsite/Makefile.sphinx

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXCONFDIR = rst
-SPHINXOPTS    = -j $(CPUS) -n -w rst_warnings -c "$(SPHINXCONFDIR)"
+SPHINXOPTS    ?= -j $(CPUS) -n -w rst_warnings -c "$(SPHINXCONFDIR)"
```

Comment:
If this change was for step 6 here: https://www.sphinx-doc.org/en/1.8/intl.html#quick-guide we probably want to make the language parameter separate from the SPHINXOPTS and then get added to it.  Something similar to:

``` Makefile
LANGOPTS ?= -l en_US
SPHINXOPTS = -j $(CPUS) -n -w rst_warnings -c "$(SPHINXCONFDIR)"  $(LANGOPTS)
```

and then on the command line:
```
make LANGOPTS='-l pt_BR -l es_MX html'
```